src/views/Workspace/Cards/AddFindPlayground.jsx

```diff
@@ -0,0 +1,394 @@
+    const c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a));
+    const d = R * c * 1000; // Distance in meters
+
+    return d;
+}
+
+function deg2rad(deg) {
+    return deg * (Math.PI / 180);
+}
+
+const ArrowIcon = () => (
+  <SvgIcon width="18px" height="14px" viewBox="0 0 24 17">
+      <path fillRule="evenodd"
+            d="M16.739,6.288 L10.661,0.288 C10.269,-0.100 9.636,-0.095 9.248,
+            0.298 C8.860,0.691 8.864,1.324 9.258,1.712 L13.601,6.000 L1.963,
+            6.000 C1.410,6.000 0.963,6.447 0.963,7.000 C0.963,7.553 1.410,
+            8.000 1.963,8.000 L13.601,8.000 L9.258,12.288 C8.864,12.676 8.861,
+            13.309 9.248,13.702 C9.444,13.900 9.702,14.000 9.960,14.000 C10.213,
+            14.000 10.467,13.904 10.661,13.712 L16.739,7.712 C16.929,
+            7.524 17.037,7.268 17.037,7.000 C17.037,6.732 16.930,6.477 16.739,
+            6.288 Z"
+      />
+  </SvgIcon>
+);
+
+const mapStateToProps = state => ({
+    playgrounds: getAllPlaygrounds(state),
+    googleMapsKey: getGoogleMapsKey(state),
+});
+
+
+const PlacesWithStandaloneSearchBox = props => (
+  <PlacesWithStandaloneSearchBoxImpl
+    {...props}
+    googleMapURL={`https://maps.googleapis.com/maps/api/js?key=${props.googleMapsKey}&v=3.exp&libraries=places`}
+  />
+);
+
+const PlacesWithStandaloneSearchBoxImpl = compose(
+  withProps({
+      loadingElement: <div style={{ height: `100%` }} />,
+      containerElement: <div style={{ height: `400px` }} />,
+  }),
+  lifecycle({
+      componentWillMount() {
+          const refs = {};
+
+          this.setState({
+              places: [],
+              onSearchBoxMounted: ref => {
+                  refs.searchBox = ref;
+              },
+              onPlacesChanged: () => {
+                  const places = refs.searchBox.getPlaces();
+
+                  this.props.getResults(places[0]);
+
+                  this.setState({
+                      places,
+                  });
+              },
+          })
+      },
+  }),
+  withScriptjs
+)(props =>
+  <div data-standalone-searchbox="" className={props.searchBar}>
+      <StandaloneSearchBox
+        ref={props.onSearchBoxMounted}
+        bounds={props.bounds}
+        onPlacesChanged={props.onPlacesChanged}
+      >
+          <TextField
+            type="text"
+            variant="outlined"
+            name="search"
+            className={props.classes.textInput}
+            label="Zoek in je plaats"
+            autoFocus
+            placeholder="e.g. 9402 AS"
+            InputLabelProps={{
+                classes: {
+                    root: props.classes.cssLabel,
+                    focused: props.classes.cssFocused,
+                }
+            }}
+            InputProps={{
+                endAdornment:
+                  <InputAdornment disablePointerEvents position="end">
+                      <Search color="disabled" />
+                  </InputAdornment>,
+            }}
+            inputProps={{
```

Comment:
this duplicate the inputProps properties which generates a warning (and thus blocks the build).
Please fix.

Reply:
Oh, actually both of them (with a different case of the first letter) is important for the correct displaying. But I'll try to find a solution.

Reply:
Done